tests/python/relax/test_expr.py

```diff
@@ -25,13 +25,12 @@
 def test_var() -> None:
     v0 = rx.Var("v0")
     assert v0.name_hint == "v0"
-    assert v0.shape_ is None
     assert v0._checked_type_ is None
     assert v0.struct_info_ is None
     shape = [54, 96]
     v1 = rx.Var("v1", R.Tensor(shape, "float32"))
     assert v1.name_hint == "v1"
-    for s0, s1 in zip(v1.shape_, shape):
+    for s0, s1 in zip(v1.struct_info.shape, shape):
```

Comment:
Use `strict=True` to ensure they have the same length. (Though in this specific case it is very unlikely they have different lengths.)
```suggestion
    for s0, s1 in zip(v1.struct_info.shape, shape, strict=True):
```

Reply:
@MasterJH5574 Thanks for the great suggestion. However, `strict` is introduced by Python 3.10. Can we remove it to pass this test in older Python env?

Reply:
Oh sorry I didn’t know this.. Googled told me that and..

In this case such a check is not necessary actually. I think we can just safely remove this 🤦